osf/models/node.py

```diff
@@ -438,6 +438,11 @@ def is_preprint_orphan(self):
         return False
 
     @property
+    def has_published_preprint(self):
+        PreprintService = apps.get_model('osf.PreprintService')
+        return PreprintService.objects.filter(node_id=self.id, is_published=True).exists()
```

Comment:
Always use the `related_name` (`preprints` in this case) for relationships when possible. Doing so allows Django to use queryset caching.